tracer/test/Datadog.Trace.Tests/Snapshots/PublicApiTests.PublicApiHasNotChanged.verified.txt

```diff
@@ -243,40 +243,53 @@ namespace Datadog.Trace
         public const string TraceId = "x-datadog-trace-id";
         public const string TracingEnabled = "x-datadog-tracing-enabled";
     }
+    public interface IScope : System.IDisposable
+    {
+        Datadog.Trace.ISpan Span { get; }
+        void Close();
+    }
+    public interface ISpan : System.IDisposable
+    {
+        Datadog.Trace.ISpanContext Context { get; }
+        bool Error { get; set; }
+        string OperationName { get; set; }
+        string ResourceName { get; set; }
+        string ServiceName { get; set; }
+        ulong SpanId { get; }
+        ulong TraceId { get; }
+        string Type { get; set; }
+        void Finish();
+        void Finish(System.DateTimeOffset finishTimestamp);
+        string GetTag(string key);
+        void SetException(System.Exception exception);
+        Datadog.Trace.ISpan SetTag(string key, string value);
+    }
     public interface ISpanContext
     {
         string ServiceName { get; }
         ulong SpanId { get; }
         ulong TraceId { get; }
     }
+    public interface ITracer
+    {
+        Datadog.Trace.IScope ActiveScope { get; }
+        Datadog.Trace.Configuration.ImmutableTracerSettings Settings { get; }
+        Datadog.Trace.IScope StartActive(string operationName, Datadog.Trace.ISpanContext parent = null, string serviceName = null, System.DateTimeOffset? startTime = default, bool ignoreActiveScope = false, bool finishOnClose = true);
+    }
     public enum SamplingPriority
     {
         UserReject = -1,
         AutoReject = 0,
         AutoKeep = 1,
         UserKeep = 2,
     }
-    public class Scope : System.IDisposable
+    public class Scope : Datadog.Trace.IScope, System.IDisposable
```

Comment:
I thought the plan was to make `Scope`/`Span` internal and only expose `IScope`/`ISpan`.

Reply:
This PR made sure the public API only passed `ISpan`/`IScope` objects. It seems like a good next step to make the implementation types internal